### ISSUE TITLE
AAP Config as Code Office Hours Agenda - Aug 3rd 2023

### ISSUE THREAD

**Issue by sean-m-sullivan** [redhat-cop/automate_tower_genie_collections#649]
# AAP Config as Code Office Hours
## What

After attending other Office Hours of various groups, we decided to host one of our own. 
The goal is to host a regular time for the AAP Config as Code contributors to be available for your folks in the Community, so we are happy to announce a new regular video meeting.

## Where & When

The meeting will be held on Thursday, Aug 3, 2023 at [1300 UTC](https://dateful.com/time-zone-converter?t=13:00&tz=UTC)

-   [Google Meet](https://meet.google.com/npj-fyzv-oyo)
-   Via Phone PIN: 621 302 239 [Guide](https://support.google.com/meet/answer/9518557)

This meeting is held the first Thursday of the month, at [1300 UTC](https://dateful.com/time-zone-converter?t=13:00&tz=UTC)

## How

Add one topic per comment in this GitHub issue

If you don't have a GitHub account, jump on [#aap_config_as_code:ansible.com](https://matrix.to/#/#aap_config_as_code:ansible.com) on Matrix and we can add the topic for you

## Talk with us

As well as the quatri-weekly video meeting you can join the Community (inc development team) on Matrix Chat.

- Matrix: [#aap_config_as_code:ansible.com](https://matrix.to/#/#aap_config_as_code:ansible.com) (recomended)

See you soon!

## Links
[Feb 15th Meeting 2023](https://github.com/redhat-cop/controller_configuration/issues/475)
[April 12th Meeting 2023](https://github.com/redhat-cop/controller_configuration/issues/568)
[May 11th Meeting 2023](https://github.com/redhat-cop/controller_configuration/issues/596)
[June 8th Meeting 2023](https://github.com/redhat-cop/controller_configuration/issues/600)
[July 6th Meeting 2023](https://github.com/redhat-cop/controller_configuration/issues/638)

**Comment by repcsi** [comment]
Hello All,

I will try to attend tomorrow, and I would like to add the topic of release schedule of this project so I can understand when bugfixes and new features will hit the official galaxy collection.

Also adding this one as well as it might be closely related:
I'm using infra/controller_config but what's the difference between:
https://galaxy.ansible.com/redhat_cop/controller_configuration
and
https://galaxy.ansible.com/infra/controller_configuration ?
Seems like the redhat_cop one is lagging behind in version quite a bit despite having way more download hits.

Thank you.

**Comment by sean-m-sullivan** [comment]
@repcsi canaddress this tomorrow as well, but we Announced that we were retiring the redhat_cop namespace 10 months ago, and stopped publishing 4 months ago, we moved to the infra namespace, so it would be the same on https://console.redhat.com/ansible/automation-hub/infra and on galaxy. We did this across our suite and updated the links as well. Something similar will be done with the automation hub collection as it has recently moved repos as well.
